CMakeLists.txt

```diff
@@ -132,6 +136,8 @@ target_compile_definitions(${PROJECT_NAME_PLAP} PRIVATE
     WIN32_LEAN_AND_MEAN
     HAVE_CONFIG_H)
 
+set_target_properties(${PROJECT_NAME_PLAP} PROPERTIES LINK_FLAGS " /ignore:4302 ")
```

Comment:
again, please document what "4302" means

Reply:
Just a question, but why are we using `set_target_properties` instead of the dedicated options?

Reply:
Changed to `target_link_options`.